services/discordService.js

```diff
@@ -103,15 +103,51 @@ const setUserDiscordNickname = async (userName, discordId) => {
     };
   } catch (err) {
     logger.error("Error in updating discord Nickname", err);
-    throw err;
+    throw new Error(err);
```

Comment:
Why this change?

Reply:
This is just a chore change for consistent error type and is useful in better debugging and error handling.